prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/components/FrankLloydRoot.jsx

```diff
@@ -34,7 +34,7 @@ export function FrankLloydRoot({ children, config }) {
           { rel: 'icon', href: 'https://sample-cdn.frank/favicon.ico' },
         ]}
       />
-      <pre className="value-provided-from-config">{ config.get('someApiUrl') }</pre>
+      <pre suppressHydrationWarning={true} className="value-provided-from-config">{ config.get('someApiUrl') }</pre>
```

Comment:
are we not able to hydrate this properly for some reason?

Reply:
the tests validate different values on client and ssr so we deliberately cause a mismatch, react 17 was happy with the react 18 not so much

Reply:
curious about those values, why do we validate different values on client and server?

Reply:
It validates that the client state config is different from the server state config.